packages/datagovmy-ui/src/components/Section/index.tsx

```diff
@@ -69,7 +69,7 @@ const Section: ForwardRefExoticComponent<SectionProps> = forwardRef(
                       {description}
                     </p>
                   ) : (
-                    <div>{description}</div>
+                    description
```

Comment:
does this impact UI in on other page ?

Reply:
@leneherng

Reply:
I checked all the `Section` descriptions that aren't a string/array and I think I've fixed the only 2 that was impacted, which was `KTMB` & `DataCatalogue Show`